frame/bounties/src/lib.rs

```diff
@@ -510,6 +535,11 @@ pub mod pallet {
 
 			Bounties::<T>::try_mutate_exists(bounty_id, |maybe_bounty| -> DispatchResult {
 				let mut bounty = maybe_bounty.as_mut().ok_or(Error::<T>::InvalidIndex)?;
+
+				// Ensure no active child-bounties before processing the call.
+				ensure!(T::ChildBountyManager::child_bounties_count(bounty_id) == 0, 
+					Error::<T>::RequireNoActiveChildBounty);
```

Comment:
Seems to me we'll need to run benchmarks for bounties as well.

Reply:
Done. The new weights for bounties pallet have been added, and they reflect additional storage reads from the child bounties pallet.